subgraphs/badger-dao/.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: Build
```

Comment:
Don't you need to change into the proper directory?  Remember that your subgraph is now in subgraphs/badger-dao.